llvm/lib/Target/AMDGPU/AMDGPUISelDAGToDAG.cpp

```diff
@@ -2389,13 +2389,15 @@ bool AMDGPUDAGToDAGISel::isCBranchSCC(const SDNode *N) const {
   if (VT == MVT::i32)
     return true;
 
+  const auto *ST = static_cast<const GCNSubtarget *>(Subtarget);
```

Comment:
You don't need this at all, just use `Subtarget` instead of `ST`.

Reply:
I fixed some cases of this in a161e73fcc95

Reply:
Done :)